schema/config-linux.json

```diff
@@ -170,11 +170,14 @@
                             },
                             "period": {
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/cpu/period",
-                                "$ref": "defs.json#/definitions/uint64Pointer"
```

Comment:
Sorry, I read #233 in detail. But I think `uint64Pointer` can't satisfy the special requirements for swappiness, period and quota.
I updated PR to support `null` type for nil pointer.